edi_voxel_stock_picking_oca/models/res_partner.py

```diff
@@ -0,0 +1,22 @@
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    voxel_report_id = fields.Many2one(
+        comodel_name="ir.actions.report",
+        domain=[("report_type", "=", "qweb-xml"), ("model", "=", "stock.picking")],
+    )
+
+    def write(self, vals):
```

Comment:
Instead of this, overwrite the method `commercial_fields` and add the field name. Example:

https://github.com/odoo/odoo/blob/b9abe46c1492b09e369434e76ec8196c6b02dd19/addons/account/models/partner.py#L658-L661